File: webservice/Frikollection_Api/Frikollection_Api/DTOs/User/UserProfileDto.cs
namespace Frikollection_Api.DTOs.User
{
    public class UserProfileDto
    {
        public Guid UserId { get; set; }
        public string Username { get; set; }
        public string Password { get; set; }

        public string Email { get; set; }

        public string? Avatar { get; set; }

        public string? Nickname { get; set; }

        public string? FirstName { get; set; }

        public string? LastName { get; set; }

        public string? PhoneNumber { get; set; }

        public DateOnly? Birthday { get; set; }

        public string? Biography { get; set; }

        public DateTime? RegisterDate { get; set; }

        public DateTime? LastLogin { get; set; }
    }
}

File: mobile_desktop_app/FrikollectionMobileDesktop/composeApp/src/commonMain/kotlin/org/example/frikollection_mobile_desktop/models/user/UserProfileDto.kt
package org.example.frikollection_mobile_desktop.models.user

import kotlinx.serialization.Serializable

@Serializable
data class UserProfileDto(
    val userId: String,
    val username: String,
    val password: String,
    val email: String,
    val avatar: String?,
    val nickname: String,
    val firstName: String?,
    val lastName: String?,
    val phoneNumber: String?,
    val birthday: String?,
    val biography: String?,
    val registerDate: String?,
    val lastLogin: String?
)